recipes/coinmp/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+sh ./configure --prefix=$PREFIX --enable-static --enable-gnu-packages
```

Comment:
Not that I know, is enables searching for GNU packages, not sure about the licensing in this case.